purchase_tier_validation/tests/common.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+def setup_test_model(env, model_clses):
```

Comment:
this approach is deprecated. Can you please use https://github.com/OCA/odoo-test-helper ?

Reply:
Updated, Thanks